MinecraftClient/Commands/Sneak.cs

```diff
@@ -0,0 +1,32 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+
+namespace MinecraftClient.Commands
+{
+    public class Sneak : Command
+    {
+        private bool sneaking = false;
+        public override string CMDName { get { return "TSneak"; } }
```

Comment:
Seems like there is an extra T: `return "Sneak"`

Reply:
This was because it does what a mod does called ToggleSneak. I shortened it to TSneak but I will fix it.